src/pages/lists/status-list/index.tsx

```diff
@@ -1,78 +1,371 @@
+        justifyContent: 'space-between',
+    },
+    listItemTextWithoutIcon: {
+        marginLeft: -theme.spacing(7),
+    },
+    listItemText: {
+        marginLeft: '0px',
+    },
+    accordionContainer: {
+        maxWidth: 768,
+        paddingTop: theme.spacing(3),
+        margin: '0 auto',
+
+        [theme.breakpoints.down('sm')]: {
+            margin: `0 0 ${theme.spacing(3)}px 0`,
+            paddingTop: 0,
+        },
+    },
+    accordionRoot: {
+        marginBottom: theme.spacing(2),
+        borderRadius: '4px',
+        '&:before': {
+            display: 'none',
+        },
+        '& .MuiAccordionSummary-root': {
+            height: '48px',
+            minHeight: '48px',
+            '&.Mui-expanded': {
+                borderBottom: `1px solid ${theme.palette.divider}`,
+            },
+        },
+        [theme.breakpoints.down('sm')]: {
+            maxWidth: '100%',
+            boxShadow: 'none',
+            borderRadius: 0,
+            '&:before': {
+                display: 'none',
+            },
+            '& .MuiAccordionSummary-root': {
+                height: '48px',
+                minHeight: '48px',
+                '&.Mui-expanded': {
+                    borderBottom: `1px solid ${theme.palette.divider}`,
+                    margin: 0,
+                },
+            },
+        },
+    },
+    accordionDetails: {
+        display: 'block',
+        padding: 0,
+    },
+    toolbarRightContent: {
+        display: 'flex',
+        flexDirection: 'row',
+    },
+    listItemTitle: {
+        display: 'flex',
+        alignItems: 'center',
+        color: colors.black[500],
+        fontWeight: 400,
+        marginLeft: theme.spacing(4),
+    },
+    listItemTitleWithoutTimeStamp: {
+        marginLeft: 'auto',
+    },
+    station: {
+        fontSize: 14,
+        fontWeight: 400,
+        marginLeft: theme.spacing(4),
+        textOverflow: 'ellipsis',
+        overflow: 'hidden',
+    },
+    stationWithoutTimeStamp: {
+        marginLeft: 0,
+    },
+    location: {
+        fontSize: 12,
+        fontWeight: 400,
+    },
+    leftComponentRoot: {
+        display: 'flex',
+        flexDirection: 'column',
+        fontSize: '12px',
+    },
+    leftComponentRootWithoutIcon: {
+        marginLeft: -theme.spacing(7),
+    },
+    timeStamp: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    time: {
+        fontWeight: 700,
+        fontSize: '12px',
+    },
+    timePeriod: {
+        marginLeft: '4px',
+    },
+    rightComponentRoot: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    assignedTag: {
+        marginLeft: theme.spacing(4),
+        marginTop: 4,
+    },
+    activeTag: {
+        marginLeft: 0,
+        marginTop: 4,
+    },
+    listItemTag: {
+        marginLeft: theme.spacing(2),
+        marginRight: theme.spacing(4),
+    },
+    rightComponentChevron: {
+        color: colors.gray[500],
```

Comment:
shouldn't need this either

Reply:
Can you log a bug in our React component library repo for this? By default, the chevron (and any non-text `rightComponent`) should be `theme.palette.text.secondary`.